include/openssl/trace.h

```diff
@@ -56,7 +56,8 @@ extern "C" {
 # define OSSL_TRACE_CATEGORY_STORE              14
 # define OSSL_TRACE_CATEGORY_DECODER            15
 # define OSSL_TRACE_CATEGORY_ENCODER            16
-# define OSSL_TRACE_CATEGORY_NUM                17
+# define OSSL_TRACE_CATEGORY_REF_COUNT          18
+# define OSSL_TRACE_CATEGORY_NUM                18      /* Last one! */
```

Comment:
Maybe you could do what I *actually* suggested (replicated here):


``` suggestion
# define OSSL_TRACE_CATEGORY_REF_COUNT          17
# define OSSL_TRACE_CATEGORY_NUM                18
```

Specifically, `OSSL_TRACE_CATEGORY_NUM` is the *count* of available categories, not a copy of the last one, so your added comment is incorrect.

Reply:
I changed the comment and pushed a fixup commit.  My point was to avoid anyone else repeating my original mistake which was not incrementing `OSSL_TRACE_CATEGORY_NUM` when adding a new category.  Sorry for making you folks spend time on this little issue.